modules/nw-aws-load-balancer-logs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+// * networking/understanding-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-aws-load-balancer-operator-logs_{context}"]
+= AWS Load Balancer Operator logs
+
+You can view the AWS Load Balancer Operator logs by using the `oc logs` command.
```

Comment:
I would use the following sentence structure to have a more active voice.

"Use the `oc logs` command to view the AWS Load Balancer Operator logs."

Reply:
Addressed